server/handlers.go

```diff
@@ -164,11 +141,17 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Construct a URL with all of the arguments in its query
+	connURL := url.URL{
+		RawQuery: r.Form.Encode(),
```

Comment:
I think redirecting a user to `/auth/my_connector?connector_id=my_connector` may be puzzling a bit.
Can we use only mux variables or query parameters?

Reply:
Removing connector_id from r.Form would seem to make sense here.